CMakeLists.txt

```diff
@@ -1,64 +1,70 @@
 cmake_minimum_required(VERSION 3.5)
-include(GNUInstallDirs)
+project(qualisys_cpp_sdk)
 
 option(BUILD_EXAMPLES "Build examples" OFF)
 
-project(qualisys_cpp_sdk)
+add_library(${PROJECT_NAME}
+        Markup.cpp
+        Network.cpp
+        RTPacket.cpp
+        RTProtocol.cpp)
+
+target_include_directories(${PROJECT_NAME} PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}>)
 
-add_library(qualisys_cpp_sdk
-  Markup.cpp
-  Network.cpp
-  RTPacket.cpp
-  RTProtocol.cpp
-)
-target_include_directories( ${PROJECT_NAME} 
-    PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-)
-target_link_libraries( ${PROJECT_NAME}
-    PRIVATE $<$<STREQUAL:"${CMAKE_CXX_COMPILER_ID}","MSVC">:ws2_32.lib>
-)
-# Enable C++11
+target_link_libraries(${PROJECT_NAME}
+        PRIVATE $<$<STREQUAL:"${CMAKE_CXX_COMPILER_ID}","MSVC">:ws2_32.lib>)
```

Comment:
This can be made public instead (and the note in the readme can then be removed).

Reply:
Oh yeah, that makes sense. Updated.